Rasa_Bot/actions/actions_profile_creation.py

```diff
@@ -0,0 +1,654 @@
+        if last_utterance != 'utter_ask_profile_creation_conf_9_slot':
+            return {"profile_creation_conf_9_slot": None}
+
+        if not validator.validate_number_in_range_response(n_min = 0, n_max = 10, 
+                                                           response = value):
+            dispatcher.utter_message(response="utter_please_answer_0_to_10")
+            return {"profile_creation_conf_9_slot": None}
+
+        return {"profile_creation_conf_9_slot": int(value)}
+    
+
+class ValidateProfileCreationConf10Form(FormValidationAction):
+    def name(self) -> Text:
+        return 'validate_profile_creation_conf_10_form'
+
+    def validate_profile_creation_conf_10_slot(
+            self, value: Text, dispatcher: CollectingDispatcher,
+            tracker: Tracker, domain: Dict[Text, Any]) -> Dict[Text, Any]:
+        # pylint: disable=unused-argument
+        """Validate profile_creation_conf_10_slot"""
+
+        last_utterance = get_latest_bot_utterance(tracker.events)
+        if last_utterance != 'utter_ask_profile_creation_conf_10_slot':
+            return {"profile_creation_conf_10_slot": None}
+
+        if not validator.validate_number_in_range_response(n_min = 0, n_max = 10, 
+                                                           response = value):
+            dispatcher.utter_message(response="utter_please_answer_0_to_10")
+            return {"profile_creation_conf_10_slot": None}
+
+        return {"profile_creation_conf_10_slot": int(value)}
+
+
+class ProfileCreationSetConfLowHighSlot(Action):
+    """Set conf low high slot"""
+
+    def name(self):
+        return "profile_creation_set_conf_low_high_slot"
+
+    async def run(self, dispatcher, tracker, domain):
+        
+        
+        conf_slots = ["profile_creation_conf_10_slot", "profile_creation_conf_9_slot",
+                      "profile_creation_conf_8_slot", "profile_creation_conf_7_slot",
+                      "profile_creation_conf_6_slot", "profile_creation_conf_5_slot",
+                      "profile_creation_conf_4_slot", "profile_creation_conf_3_slot",
+                      "profile_creation_conf_2_slot", "profile_creation_conf_1_slot"]
+        
+        # Find the conf slot we just set
+        conf = -1
+        for conf_slot in conf_slots:
+            conf = tracker.get_slot(conf_slot)
+            if conf > -1:
+                break
+        
+        low_high = 0
+        if conf > 1:
+            low_high = 1
+            
+        return [SlotSet("profile_creation_conf_low_high_slot", low_high)]
+    
+
+class ProfileCreationSaveToDB(Action):
+    """Save profile creation data to database"""
+
+    def name(self):
+        return "profile_creation_save_to_db"
+
+    async def run(self, dispatcher, tracker, domain):
+    
+        user_id = tracker.current_state()['sender_id']
+        
+        # Get confidence slots
+        conf = [tracker.get_slot("profile_creation_conf_" + str(i) + "_slot") for i in range(1, 11)]
```

Comment:
if we the `conf_slots ` list, this can be reformulated as:
```suggestion
        conf = [tracker.get_slot(slot) for slot in conf_slots]
```